##[pylyzer] failed /home/dani/advent-of-code-2024/day13/venv/lib/python3.12/site-packages/scipy/integrate/_odepack_py.py 1734628983 11150
.__all__: global::List!({"ODEintWarning", "odeint"}, 2)
.np = pyimport "__init__"
.__init__ = pyimport "__init__"

.___v_desugar_1 = pyimport "__init__"

._odepack: Never
.___v_desugar_2 = pyimport "<failure>"
.<failure> = pyimport "<failure>"
.copy: Never
.warnings = pyimport "<failure>"


.ODEintWarning: ClassType
.ODEintWarning <: global::Warning

._msgs: global::Dict!({{1}: {"Nothing was done; the integration time was 0."}, {2}: {"Integration successful."}, Int: {"Internal workspace insufficient to finish (internal error)."}})
.odeint: (func: Obj, y0: Obj, t: Obj, args: Obj := Obj, Dfun: Obj := Obj, col_deriv: Obj := Obj, full_output: Bool := Bool, ml: Obj := Obj, mu: Obj := Obj, rtol: Obj := Obj, atol: Obj := Obj, tcrit: Obj := Obj, h0: Obj := Obj, hmax: Obj := Obj, hmin: Obj := Obj, ixpr: Obj := Obj, mxstep: Obj := Obj, mxhnil: Obj := Obj, mxordn: Obj := Obj, mxords: Obj := Obj, printmessg: Bool := Bool, tfirst: Obj := Obj) -> Never
